project/Dependencies.scala

```diff
@@ -29,7 +29,7 @@ object Dependencies {
 
   val scala212Version = "2.12.14"
   val scala213Version = "2.13.6"
-  val scala3Version = "3.0.1-RC1"
+  val scala3Version = "3.0.2-RC1"
```

Comment:
So, we just track the patched / RC tightly? I guess no problem with that as long as our Scala 3 build is not complete.